ui/app/src/components/workspaces/WorkspaceServiceItem.tsx

```diff
@@ -49,6 +49,23 @@ export const WorkspaceServiceItem: React.FunctionComponent<WorkspaceServiceItemP
     getData();
   }, [apiCall, props.workspace.id, props.workspace.properties.app_id, props.workspaceService, workspaceServiceId]);
 
+  const updateUserResource = (u: UserResource) => {
+    let ur = [...userResources];
+    let i = ur.findIndex((f: UserResource) => f.id === u.id);
+    ur.splice(i, 1, u);
+    setUserResources(ur);
+  }
+
+  const removeUserResource = (u: UserResource) => {
+    let ur = [...userResources];
+    let i = ur.findIndex((f: UserResource) => f.id === u.id);
+   
+    console.log(ur[i]);
```

Comment:
We may not want to leave so much console logging in as will make it more difficult for other devs to debug

Reply:
yep my bad, meant to take this out